api/src/main/scala/dcos/metronome/api/v1/models/package.scala

```diff
@@ -249,7 +249,7 @@ package object models {
   implicit lazy val QueuedTaskInfoWrites: Writes[QueuedJobRunInfo] = new Writes[QueuedJobRunInfo] {
     //    output of queue is only runid
     override def writes(runInfo: QueuedJobRunInfo): JsValue = Json.obj(
-      "runId" -> JsString(runInfo.id.toString()))
```

Comment:
I think that we have to return jobId and runId then (as two fields). WDYT?

Because for the info to be useful for the users, they need enough informations to get more info from this endpoint
/v1/jobs/{jobId}/runs/{runId}

Reply:
we already do that... the return is { jobid: "xyz, runs: [{ runid: "test}]}

Reply:
oh, you're absolutely right! then please also change queue_job_list.json or other places where the old thing can be

Reply:
also please create a lazy val runId on QueuedJobRunInfo that would do this `runInfo.id.path.last` and then just call this val from here

Reply:
yep yep... good eyes and thought!